ledger/participant-integration-api/src/main/scala/platform/store/FlywayMigrations.scala

```diff
@@ -31,6 +31,7 @@ private[platform] class FlywayMigrations(
       .configure()
       .locations((locations(enableAppendOnlySchema, dbType) ++ additionalMigrationPaths): _*)
       .dataSource(dataSource)
+      .connectRetries(600)
```

Comment:
deferring waiting for connectivity to Flyway.
this would do exponential backoff waiting 1 2 4 8 16 ... seconds between retries
Flyway is called almost every case at startup: exception sandboxnext reset triggered restart

Reply:
[Flyway says](https://flywaydb.org/documentation/configuration/parameters/connectRetries) it always waits 1sec between retries. Waiting for 10min before declaring there is no connectivity to the database seems a bit long, no?

Reply:
as discussed offline:
* Flyway does exponential backoff now (despite docs) https://github.com/flyway/flyway/commit/37675a0e9c09b301f58142986924ad64cc0834b4
* lowered it to 10 ~ 1023 sec